src/Data/API/Tools/CBOR.hs

```diff
@@ -170,15 +170,15 @@ encodeUTCTime :: UTCTime -> Encoding
 #if MIN_VERSION_aeson(0,8,1)
 encodeUTCTime = encodeUTCTime'
 #else
-encodeUTCTime t = encodeTag 0 <> encode (formatTime defaultTimeLocale fmt t)
+encodeUTCTime t = encode (parseUTC_ $ formatTime defaultTimeLocale fmt t)
  where
    fmt = "%FT%T." ++ take 3 (formatTime defaultTimeLocale "%q" t) ++ "Z"
 #endif
 
 -- | Encode a UTCTime to match 'ToJSON' instances generated by
 -- api-tools, which never include fractions of a second.
 encodeUTCTime' :: UTCTime -> Encoding
-encodeUTCTime' t = encodeTag 0 <> encode (mkUTC' t)
+encodeUTCTime' = encode
```

Comment:
The comment here is now rather misleading. We're now CBOR-encoding `UTCTime` "normally" and dealing with the correspondence with JSON elsewhere. I'd just inline and get rid of `encodeUTCTime'`.